tools/memap.py

```diff
@@ -839,18 +842,48 @@ def compute_report(self):
             for k in self.sections:
                 self.subtotal[k] += mod[k]
                 self.subtotal[k + '-delta'] += mod[k]
+            for k in self.other_sections:
```

Comment:
I suspect is it somewhere here

Travis tools tests are failing, ref https://travis-ci.org/ARMmbed/mbed-os/jobs/659195929

`KeyError: '.rodata'` . Means `k` is `.rodata` and does not exist ?

Reply:
thanks for reviewing,
self.sections contains all section that were found in the mapfile, it is not mandatory that all sections are present. I think the mapfile for testing does not contain the rodata section.
I must check why this spliting in different groups was choosen, it looks like is necessary for the different toolchains. But this was the problem where the difference came from, with counting all .text sections you do not get the correct figure for the flashsize, other sections are also placed in the flash (dependent on linkerfile).

Reply:
Test should bechanged or this code. We shall be catching this in case memap contains not valid sections (not certain it would fail anywhere else during the build, some symbols we use in the code).

Reply:
I will try to find some more documentation on the generated sections, I have tested also an output for a K64F target and there are .flash_config and .interrupts that are placed also into flash.
It will be difficult to find invalid sections, there may be even more sections when the linkerfile is changed.